### PR TITLE
Add user model

### PR DESCRIPTION
**Pull request by tochman**
PT Link https://www.pivotaltracker.com/story/show/113903681
- Generated  `User`
- Generated Godmin resource for `User` with filter and `destroy` batch action
- Added unit specs
- WIP CRUD scenarios in cucumber

capybara-20160220095140860422165

capybara-20160220095359112852053

capybara-201602200953586120942072

capybara-201602200954005790568784

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
```
Comment (sampritipanda) [inline features/step_definitions/basic_steps.rb]: Can you add the save_and_open_screenshot command here as well? That is really helpful in debugging.

Comment (tochman) [inline features/step_definitions/basic_steps.rb]: Good idea @sampritipanda

**Comment by sampritipanda** [comment]
Does the `user_name` field store the person's name of his login ID?

If it stores the name, then I think just `name` is more appropriate in this case.

**Comment by tochman** [comment]
We have not come that far yet. This story is about creating the model only. We’ll probably change that later.

**Comment by sampritipanda** [comment]
Ok, fair enough. It's just that `user.user_name` feels a bit redundant/repetitive.

**Comment by tochman** [comment]
I added poltergeist and marked the whole feature with `@javascript` and then screenshot generation works. It is very useful. I wonder if there is a better way to set that up though. It’s a separate chore I think. Want to help me merge this in @sampritipanda  ?

File: features/godmin/users.feature [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@javascript
```
Comment (sampritipanda) [inline features/godmin/users.feature]: Should we keep the javascript tag here even though it's not needed?

Comment (tochman) [inline features/godmin/users.feature]: Good question. Probably not. ;-)

**Pull request merged**